docs/source/quick_start/examples/update_dataset.rst

```diff
@@ -104,15 +104,15 @@ Overwrite uploaded data to dataset.
    :start-after: """Update data/ overwrite dataset"""
    :end-before: """"""
 
-The default value of `skip_uploaded_files` is false, use it to overwrite uploaded data.
+The default value of ``skip_uploaded_files`` is ``False``, and you can set it to ``True`` to overwrite uploaded data.
 
 .. note::
    The segment name and data name are used to identify data,
-   which means if two data's segment names and data names are the same,
-   then they will be regarded as one data.
+   so if you upload a data whose segment name and data name are the same with certain data uploaded,
+   and ``skip_uploaded_files`` is  ``True``, then the former one will be replaced.
```

Comment:
do not use the second person in docs. 
do not need to add ``skip_uploaded_files`` here. Even if you add it, it shouldn't be "True"